src/components/WorkoutBoard.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import {Display, Root, CardGrid } from './styled' 
+import { firestore } from '../services/firebase';
+
+interface WorkoutCardDetail {
+  userId: string,
+  workoutScore?: number
+};
+
+export default function WorkoutBoard() {
+  const [userScore, setUserScore] = useState<WorkoutCardDetail|undefined>(); 
+   
+  useEffect(() => { 
```

Comment:
This will make the app run into an infinite render loop because you haven't specified the dependency list for `useEffect` - i.e. it will call out to fetch data from firestore, update the state which will trigger another render of the component, and call out to fetch data from firestore again and so forth.